src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -55,12 +55,7 @@
 import org.zendesk.client.v2.model.UserRelatedInfo;
 import org.zendesk.client.v2.model.dynamic.DynamicContentItem;
 import org.zendesk.client.v2.model.dynamic.DynamicContentItemVariant;
-import org.zendesk.client.v2.model.hc.Article;
-import org.zendesk.client.v2.model.hc.ArticleAttachments;
-import org.zendesk.client.v2.model.hc.Category;
-import org.zendesk.client.v2.model.hc.Section;
-import org.zendesk.client.v2.model.hc.Subscription;
-import org.zendesk.client.v2.model.hc.Translation;
+import org.zendesk.client.v2.model.hc.*;
```

Comment:
I am not sure about the formatter configured in this project but you can probably discard such change and keep the full list of imports